pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -127,6 +127,24 @@ func (a *Bootstrap) Generate(dependencies asset.Parents) error {
 	if err != nil {
 		return err
 	}
+
+	// Check for optional platform specific directories
+	platform := installConfig.Config.Platform.Name()
+	platformFilePath := fmt.Sprintf("bootstrap/%s/files", platform)
+	if _, err := os.Stat(platformFilePath); !os.IsNotExist(err) {
```

Comment:
This check does not honor the `OPENSHIFT_INSTALL_DATA` environment variable.

Reply:
Thanks, I switched to using data.Assets in the latest revision which I think should resolve this